r3.asl

```diff
@@ -0,0 +1,26 @@
+// mars robot 1
+
+/* Initial beliefs */
+
+at(P) :- pos(P,X,Y) & pos(r3,X,Y).
+
+
+/* Initial goal */
+
+!explore(slots).
+
+/* Plans */
+
++!explore(slots) : true
+   <- randMove(slot);
+   	  maybePoop(garb);
+      !explore(slots).
+	  
+//+!randmove(slot): true
```

Comment:
might want to delete commented code

Reply:
Is your approval to merge contingent on deleting commented code? Is this pull request approved?